### PR TITLE
Add eul Latest

### PR DESCRIPTION
**Pull request by voltapp2**
After making all changes to the cask:

- [ x ] `brew cask audit --download {{cask_file}}` is error-free.
- [ x ] `brew cask style --fix {{cask_file}}` reports no offenses.
- [ x ] The commit message includes the cask’s name and version.

Additionally, if **adding a new cask**:

- [ x ] Named the cask according to the [token reference].
- [ x ] `brew cask install {{cask_file}}` worked successfully.
- [ x ] `brew cask uninstall {{cask_file}}` worked successfully.
- [ x ] Checked there are no [open pull requests] for the same cask.
- [ x ] Checked the cask was not already refused in [closed issues].
- [ x ] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

File: Casks/eul.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+cask 'eul' do
+  version :latest
```
Comment (commitay) [inline Casks/eul.rb]: `0.3.4`

File: Casks/eul.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+  sha256 :no_check
```
Comment (commitay) [inline Casks/eul.rb]: Add the `sha256`

File: Casks/eul.rb [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+
+  url 'https://eul.im/mac'
```
Comment (commitay) [inline Casks/eul.rb]: `"https://github.com/eul-im/eul/releases/download/v#{version}/eul_mac.zip"`

**Review by commitay — CHANGES_REQUESTED** [review]

**Comment by eul-im** [comment]
Thanks for the fixes, @commitay 

Just one question. I'm planning to release updates very often. Will I have to submit pull requests for every single update?

**Comment by eul-im** [comment]
Thanks, I was about to add appcast :)

**Comment by commitay** [comment]
Yes, every update will need a PR. [We have a script that you can use](https://github.com/caskroom/homebrew-cask/blob/master/CONTRIBUTING.md#updating-a-cask) or another contributor may update it themselves.

**Comment by eul-im** [comment]
All right. I will also implement auto-updates, so a PR for every single update will not be I necessary I guess.

**Review by commitay — APPROVED** [review]

**Review by victorpopkov — APPROVED** [review]

**Comment by victorpopkov** [comment]
Thank you for your contribution!

**Pull request merged**